symbols/arm9.yml

```diff
@@ -726,6 +726,67 @@ arm9:
         r0: program position info
         r1: format
         ...: variadic
+    - name: OpenAllPackFiles
+      address:
+        EU: 0x200C364
+      description: Open the 6 files at PACK_FILE_PATHS_TABLE into PACK_FILE_OPENED. Called during game initialisation.
```

Comment:
Put an explicit "No params." where the param list would go to indicate that there are no parameters. Leaving it off implies the parameter list is unknown.